ui.apps/src/main/content/jcr_root/apps/acs-commons/authoring/sites-copy-publishurl/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[cq.authoring.editor]"
```

Comment:
Do we have to worry about this JS being injected even when this feature isnt being used?

Reply:
Added a wrapper client library, so that the `sites-copy-publish-url.js` is not enabled by default.